contents/handbook/people/onboarding.md

```diff
@@ -18,16 +18,12 @@ Charles will create the contract needed, depending on who is joining. Only James
 
 ### US Team Member Checklist
 
-<input type="checkbox"/>  Use [Clerky](https://app.clerky.com). Choose 'New' and select 'Employee Agreement' and then 'Offer Letter (with equity compensation) and CIIA Agreement — Employee Agreement'. Follow the instructions. <br>
-<input type="checkbox"/>  Enter 'Delaware' for State of Incorporation. <br>
-<input type="checkbox"/>  Enter 'Stock Options' for Type of Restricted Stock  <br>
-<input type="checkbox"/>  Fill out the sections for Description of Excluded IP and Restrictive Prior Agreements if relevant, otherwise ignore <br>
-<input type="checkbox"/>  Enter 'State of Delaware' for Governing Law Jurisdiction <br>
-<input type="checkbox"/>  Complete the rest of the details and send to the team member for signing <br>
+<input type="checkbox"/>  Create a contract using the Google Docs template in the Legal Docs shared drive <br>
```

Comment:
A link would be helpful

Reply:
Good point, done - also added a link for the UK contract template while I was at it